services/docker-compose.yml

```diff
@@ -195,6 +195,20 @@ services:
       - INVITATIONS_PASSWORD=${INVITATIONS_PASSWORD}
       - LOG_FORMAT_LOCAL_DEV_ENABLED=${LOG_FORMAT_LOCAL_DEV_ENABLED}
 
+  payments:
+    image: ${DOCKER_REGISTRY:-itisfoundation}/payments:${DOCKER_IMAGE_TAG:-latest}
+    init: true
+    hostname: "{{.Node.Hostname}}-{{.Service.Name}}-{{.Task.Slot}}"
+    networks:
+      - default
```

Comment:
who does this guy talk to actually? (microservices/ outside?)

Reply:
webserver, postgres and RUT